tests/README.md

```diff
@@ -0,0 +1,173 @@
+      - [Truth Branches](#truth-branches)
+      - [Results vs Storage buckets](#results-vs-storage-buckets)
+  - [:hammer_and_pick: Testing Process](#hammer_and_pick-testing-process)
+    - [Scala Framework](#scala-framework)
+      - [Command Format](#command-format)
+    - [What does the wrapper workflow do?](#what-does-the-wrapper-workflow-do)
+  - [:mag_right: Working with tests locally](#mag_right-working-with-tests-locally)
+    - [Creating a new test](#creating-a-new-test)
+    - [Running your new test](#running-your-new-test)
+
+## :dna: Overview 
+
+This README serves as a summary of the testing process for the pipelines that exists in WARP.
+
+### Why test?
+
+Downstream users of these pipelines expect robustness and consistency. When a change is made to a pipeline it is critical to ensure that the change does not affect the scientific validity of pipeline outputs.
+
+## :page_with_curl: Terms and Concepts
+
+#### Main Workflow/Pipeline
+
+Every pipeline that exists in the [pipelines/](../pipelines/) directory of WARP can be though of as a "main" workflow. These are the components that are actually being tested. 
+
+#### Wrapper Workflow
+
+Every main pipeline has an accompanying "wrapper" workflow in the [verification/test-wdls/](../verification/test-wdls/) directory. These wrapper workflows are designed to run the main workflow and pass its outputs along to the validation workflow.
+
+#### Validation Workflow
+
+The validation workflow takes the output of a completed pipeline and compares it against a known truth set for that pipeline. As with the wrapper workflow, each main workflow must also have a [validation workflow](../verification/).
+
+#### Plumbing vs Scientific data
+
+Each main workflow should have two sets of test data, plumbing and scientific. The plumbing test data is typically a set of downsampled inputs designed to test that the pipeline runs to completion without error. The scientific test data is a set of full-sized inputs designed to thoroughly test the pipeline.
+
+These test inputs exist alongside each pipeline in their respective `test_inputs` directory.
+
+#### Truth Branches
+
+The testing framework has a concept of "truth branches". This allows users to define a branch of truth output to run against.
+
+In most cases the branch being run against is "master" which corresponds to a google storage bucket containing the agreed upon output of the pipeline as it exists in the "master" git branch.
+
+#### Results vs Storage buckets
+
+Every workflow run in cromwell has its output stored in an "execution" bucket. When running the pipeline tests it's important that we store this ouptut in a location that we can later reference if needed.
+
+All test workflows that are run will have their output copied to a "results" bucket -> `gs://broad-gotc-test-results/`
+
+When we want to update the "truth" branches for our pipelines we can copy that output to the respective "truth" bucket -> `gs://broad-gotc-test-storage/`
```

Comment:
I thought we had an update truth command? Is it a manual copy?

Reply:
Yeah, this is just referring to what the workflow actually does, let me try to make it a little more clear